declarations/plugins/sharing/ConsumeSharedPlugin.d.ts

```diff
@@ -72,3 +73,26 @@ export interface ConsumesConfig {
 	 */
 	strictVersion?: boolean;
 }
+/**
+ * This interface was referenced by `ConsumeSharedPluginOptions`'s JSON-Schema
+ * via the `definition` "SharedExclusionCriteriaWithFallbackVersion".
+ */
+export interface SharedExclusionCriteriaWithFallbackVersion {
+	/**
+	 * Version of the package provided by the fallback if there is one.
+	 */
+	fallbackVersion?: string;
```

Comment:
Adds a new property called `fallbackVersion`, which is only exposed to `ConsumeSharedPlugin`. It is not directly accessible to `ModuleFederationPlugin` consumers. The value of this property is the truthy value of `version` property passed to `ModuleFederationPlugin`.